packages/plugin-table/src/Table.ts

```diff
@@ -99,7 +99,7 @@ export class Table<T extends TableConfig = TableConfig> extends JWPlugin<T> {
             },
             {
                 id: 'TablePicker',
-                async render(editor: JWEditor): Promise<VNode[]> {
+                async render(editor: JWEditor): Promise<TablePickerNode[]> {
```

Comment:
Is this really useful ? I'm not complaining, it can stay there I don't mind but I'm wondering if this actually changes something meaningful somewhere.

Reply:
No but I find it clearer: it makes it clear that what is being rendered is a special node that I can then go inspect.